package.json

```diff
@@ -178,6 +178,7 @@
     "socket.io-client": "4.7.0",
     "stream-browserify": "3.0.0",
     "swiper": "8.4.2",
+    "tweetnacl": "1.0.3",
```

Comment:
why this?

Reply:
It's for signing the claim message which is done in `src/modules/message/utils/signMessageWithPrivateKey.js`